src/pages/sign/signIn/page.tsx

```diff
@@ -1,11 +1,86 @@
-import { Layout } from "antd";
+import { Button, Form, Input, Layout } from "antd";
+import { useUnit } from "effector-react";
 import { PageHeader } from "../../../shared/ui/PageHeader/PageHeader";
+import { signInAdmin, signInUser } from "./model";
+import "./page.css";
 
 const { Content } = Layout;
 
-export const SignInPage = () => (
-  <Layout>
-    <PageHeader title="Вход" />
-    <Content />
-  </Layout>
-);
+type Value = {
+  nickName: string;
+  password: string;
+};
+
+const users = [
+  { nickName: "asd", password: "123" },
+  { nickName: "qwe", password: "123" },
+  { nickName: "admin", password: "123" },
+];
+
+export const SignInPage = () => {
+  const [signInAdminFn, signInUserFn] = useUnit([signInAdmin, signInUser]);
+
+  const onFinish = (user: Value) =>
+    user.nickName === "admin" ? signInAdminFn() : signInUserFn();
+
+  return (
+    <Layout>
+      <PageHeader title="Вход" />
+      <Content className="Content--signInPage">
+        <Form
+          className="Form"
+          onFinish={onFinish}
+          autoComplete="off"
+          validateTrigger="onFinish"
+        >
+          <Form.Item
+            name="nickName"
+            rules={[
+              { required: true, message: "Введите ник!" },
+              () => ({
+                validator(_, value) {
+                  if (users.some((user) => user.nickName === value)) {
```

Comment:
why are you use promises? is it necessary here?

Reply:
antD input rules require promices